api-mock/api-mock-server.ts

```diff
@@ -111,68 +285,127 @@ app.use((req, res, next) => {
 });
 app.use(middleware());
 app.use(errorHandler());
-app.use((err, req, res, next) => {
-    log(err);
-    res.status(500).send("error");
-});
 
 app.get("/mock/ping", async (req, res, next) => {
     res.json({ ok: true });
 });
 
-app.post("/mock/reset", async (req, res, next) => {
-    Object.keys(vars).forEach((key) => (vars[key] = ""));
-    initST(req.body);
+app.post("/mock/init", async (req, res, next) => {
```

Comment:
I think `/mock/`  should become something like `/test/` because in most cases we are not actually mocking anything.

Reply:
done